master/text.html

```diff
@@ -1987,7 +1987,7 @@ <h3 id='InlineSize'>The <span class="property">'inline-size'</span> property</h3
     </tr>
     <tr>
       <th>Value:</th>
-      <td><a>&lt;length&gt;</a> | <a>&lt;percentage&gt;</a> | <a>&lt;number&gt;</a></td>
+      <td><a>&lt;length-percentage&gt;</a> | <a>&lt;number&gt;</a></td>
```

Comment:
inline-size is also specified in Logical Properties as logical width. I think this is the same thing. in that spec it is defined as  `<width>` , which links to CSS22: `<length>` | `<percentage>` | auto | inherit  (sans number, and with auto keyword). Presumably the extra number is for compatibility with presentational attributes?

Reply:
No, allowing number as a replacement for length happens automatically in all presentation attributes, see https://svgwg.org/svg2-draft/types.html#syntax. Explicitly allowing number in the CSS grammar allows it in the style property, too (this is needed for many SVG-specific properties from SVG 1). Since this is a new property, there is no legacy reason to allow plain number in the CSS property, it is probably best to harmonize with the definition in Logical Properties.